index.html

```diff
@@ -353,8 +417,11 @@ <h2>Securing JSON-LD Verifiable Credentials with SD-JWT</h2>
           to [[VC-DATA-MODEL-2.0]].
         </p>
         <p>
-          A [=conforming SD-JWT issuer implementation=] MUST use [[SD-JWT]] to secure this media type.
-	  The unsecured verifiable credential is the unencoded SD-JWT payload.
+A [=conforming SD-JWT issuer implementation=] MUST use [[[SD-JWT]]] to secure
+this media type. The unsecured [=verifiable credential=] is the input JSON
+claim set. The Issuer then converts the input JSON claim set (i.e., the
+unsecured [=verifiable credential=]) into an SD-JWT payload according to
+[[[SD-JWT]]] [[SD-JWT]].
```

Comment:
Line 424 should probably be changed from `[[[SD-JWT]]]` to a `data-cite` targeting the specific section.

Reply:
adjusted in [8c3772b](https://github.com/w3c/vc-jose-cose/pull/261/commits/8c3772bc5d3bbfea41d47a2d1e86ebfee3dde82f)